geode-core/src/test/java/org/apache/geode/internal/offheap/DataTypeJUnitTest.java

```diff
@@ -98,26 +106,33 @@ public void testDataSerializableFixedIDShort() throws IOException {
     byte[] bytes = baos.toByteArray();
     String type = DataType.getDataType(bytes);
     assertEquals(
-        "org.apache.geode.internal.DataSerializableFixedID:" + ShutdownAllResponse.class.getName(),
+        "org.apache.geode.internal.DataSerializableFixedID:" + ShutdownAllResponse.class
+            .getName(),
         type);
   }
 
   @Test
   public void testDataSerializableFixedIDInt() throws IOException, ClassNotFoundException {
-    Integer someDSFIDInt = new Integer(1);
+    assertFalse(DSFIDFactory.getDsfidmap2().containsKey(66000));
```

Comment:
It'd be nice to refactor `66000` into a `final` local variable, maybe called `testKey`, since it is used throughout the test.

Reply:
this is not necessary, because it's the same as CustId's 1. We will only use it within this test class. I only need to create a number which is bigger than short can hold.